tests/discrepancy/test_cd.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.ensemble import RandomForestClassifier
+
+from pywhy_stats import bregman, kcd
+
+seed = 12345
+rng = np.random.default_rng(seed)
+# number of samples to use in generating test dataset; the lower the faster
+n_samples = 150
```

Comment:
Generally, rather use the `flaky` decorator with a 'reasonable' corresponding assert definition instead of a fixed random seed in unit tests. Happened too often that a bug was not captured because it just worked with a concrete seed. If we just generate random data, then we would capture it if it is consistently failing. I know the concern regarding the "random fails" or runtime in stochastic tests, but this is rarely a practical issue if we set the retries to like 3.